<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="boton_volver">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_transversal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Corte Transversal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_coronal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Corte Coronal</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_sagital">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_transversal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_coronal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_sagital">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>Corte Sagital</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
